README.md

```diff
@@ -168,20 +168,20 @@ Code snippets like `var foo = "bar";` can be shown inline.
 Also, `this should vertically align` ~~`with this`~~ ~~and this~~.
 
 Code can also be shown in a block element.
-````
+```
```

Comment:
Also, according to https://help.github.com/articles/creating-and-highlighting-code-blocks/#fenced-code-blocks:

> We recommend placing a blank line before and after code blocks to make the raw formatting easier to read.

So perhaps it's a good idea to also insert a blank line before the \`\`\`?